force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -80,16 +85,25 @@ public with sharing class ClassesObjectsMethods {
      * @return A string consisting of only unique words from the input sentence.
      */
     public static String removeDuplicateWords(String sentence) {
-        // Split the sentence into words
-
-        // Create a set to store the unique words
-
-        // Add each word to the set
-
-        // Join the unique words back into a sentence
-
-        // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        List<String> wordList = sentence.split(' ', 5);
+        Set<String> wordSet = new Set<String>();
+        for (Integer i = 0; i < wordList.size(); i++) {
+            if (wordSet.contains(wordlist[i])) {
+                System.debug('set of words contains this word');
+            } else {
+                wordSet.add(wordlist[i]);
+            }
+        }
+        String newSentence = '';
+        Integer count = 0;
+        for (String word : wordSet) {
+            newSentence += word;
+            count += 1;
+            if (count < wordSet.size()) {
+                newSentence += ' ';
+            } 
+        }  
+        return newSentence;
```

Comment:
There is a Set constructor that takes a List, you can use that to remove duplicates.

Look at the String.join() method. It will be your best friend...